### PR TITLE
docs(v6): fix missing information on enums

### PR DESCRIPTION
**Pull request by fzn0x**
This PR provide the missing information on https://github.com/sequelize/website/issues/238

### CODE REVIEW

File: versioned_docs/version-6.x.x/other-topics/other-data-types.md [hunk 1]
```diff
@@ -179,16 +200,26 @@ await User.findAll({
...
-DataTypes.INET                        // INET                  PostgreSQL only
-DataTypes.MACADDR                     // MACADDR               PostgreSQL only
+| Sequelize DataType                                                                       | PostgreSQL                                                                | [MariaDB](https://mariadb.com/kb/en/geometry-types/) | [MySQL](https://dev.mysql.com/doc/refman/8.0/en/spatial-type-overview.html) | MSSQL | SQLite | Snowflake | db2 | ibmi |
+|------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|------------------------------------------------------|-----------------------------------------------------------------------------|-------|--------|-----------|-----|------|
+| [`GEOMETRY`](pathname:///api/v7/interfaces/DataTypes.GeometryDataTypeConstructor.html)   | [`GEOMETRY`](https://postgis.net/workshops/postgis-intro/geometries.html) | `GEOMETRY`                                           | `GEOMETRY`                                                                  | ❌     | ❌      | ❌         | ❌   | ❌    |
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/other-topics/other-data-types.md:209]: This should be referring to the v6 docs.

File: versioned_docs/version-6.x.x/other-topics/other-data-types.md [hunk 2]
```diff
@@ -179,16 +200,26 @@ await User.findAll({
...
+| `GEOMETRY('POLYGON')`                                                                    | `GEOMETRY(POLYGON)`                                                       | `POLYGON`                                            | `POLYGON`                                                                   | ❌     | ❌      | ❌         | ❌   | ❌    |
+| `GEOMETRY('LINESTRING')`                                                                 | `GEOMETRY(LINESTRING)`                                                    | `LINESTRING`                                         | `LINESTRING`                                                                | ❌     | ❌      | ❌         | ❌   | ❌    |
+| [`GEOGRAPHY`](pathname:///api/v7/interfaces/DataTypes.GeographyDataTypeConstructor.html) | [`GEOGRAPHY`](https://postgis.net/workshops/postgis-intro/geography.html) | ❌                                                    | ❌                                                                           | ❌     | ❌      | ❌         | ❌   | ❌    |
```
Comment (WikiRik) [inline versioned_docs/version-6.x.x/other-topics/other-data-types.md:214]: This one as well

**Review by WikiRik — COMMENTED** [review]
Will have to check if this information is actually correct for v6 later but I do already have these two comments

**Comment by fzn0x** [comment]
Hi, is there any chance you can check on this PR? @WikiRik I would like this to be merged :)

**Comment by WikiRik** [comment]
Yes, will check it out later today. Totally forgot about this one, thanks for the reminder!

**Comment by fzn0x** [comment]
> Yes, will check it out later today. Totally forgot about this one, thanks for the reminder!

Sure, thank you too

**Review by WikiRik — APPROVED** [review]

**Pull request merged**